packages/shorebird_cli/lib/src/commands/init_command.dart

```diff
@@ -71,7 +69,7 @@ Please make sure you are running "shorebird init" from within your Flutter proje
     try {
       final flavors = await Future.wait([
         _maybeGetAndroidFlavors(projectRoot.path),
-        _maybeGetiOSFlavors(projectRoot.path),
+        Future.value(ios.flavors()),
```

Comment:
I wonder if we shouldn't make ios.flavors use `async` methods instead of the `sync` ones.

I am not completely sure, but I believe that even though we are calling this wrapped in a `Future.value`, and in a `Future.wait`, it will bloc the execution and not execute android and iOS in parallel.

Either way idk if that is much of an issue anyway, but wanted to raise the wondering.

Reply:
I cleaned this up a bit to remove the `Future.wait` and have it just be:

```dart
      androidFlavors = await _maybeGetAndroidFlavors(projectRoot.path);
      iosFlavors = ios.flavors();
```